docs/content/about-iota/FAQ.mdx

```diff
@@ -99,22 +99,121 @@ For current token holders:
 - **[Staking Opportunities](./tokenomics/validators-staking.mdx):** Token holders can participate in staking or delegation to earn rewards.
 
-## 12. Will there be any migration steps for users?
+## 12. Firefly Isn't Working
+
+**Firefly for IOTA is deprecated** and no longer supports the new IOTA Mainnet network.
+
+You should switch to the [new IOTA Wallet](https://chrome.google.com/webstore/detail/iota-wallet/lfhohhmmcplfddhojblhpbblpbcoecha).
+
+- You can import your **mnemonic** or connect your **Ledger device**.
+- If the in-app updater is broken (on Windows), [download the latest version manually from Firefly](https://firefly.iota.org/).
+
+## 13. How to Migrate from Firefly/Bloom?
+
+Firefly IOTA is no longer compatible with the new IOTA Mainnet. Please migrate to the new [IOTA Wallet](https://chrome.google.com/webstore/detail/iota-wallet/lfhohhmmcplfddhojblhpbblpbcoecha).
+
+### Using a Mnemonic:
+1. Go to the new wallet.
+2. Create a new profile by importing your mnemonic.
+3. See: [Getting Started with the IOTA Wallet](https://docs.iota.org/about-iota/iota-wallet/getting-started)
+
+### Using a Ledger Device:
+1. Ensure your IOTA Ledger App is up to date:
+   - **Latest version**: 0.9.3
+   - **Ledger Nano S** (deprecated): 0.9.2
+2. Install via Ledger Live.
+3. See: [Use IOTA Wallet with a Ledger Device](https://docs.iota.org/about-iota/iota-wallet/ledger)
+
+If you're using a **Ledger Nano S**, follow this [specific guide](https://docs.iota.org/about-iota/iota-wallet/ledger).
+
+:::info
+After importing your mnemonic or connecting your Ledger, run the **Balance Finder**. This tool will help locate all your addresses. You may need to press "search" multiple times, especially if your funds are widely distributed.
+
+:::
+
+:::tip
+
+Tip: It's easier to use the Balance Finder from the expanded wallet view.
+
+:::
+
+## 14. I Have Missing Balances. Where's My Money?
+
+1. Ensure you're using the **correct mnemonic, seed, or Ledger**.
+2. Use the **Balance Finder** to search for all your addresses:
+   - [How to use Balance Finder](https://docs.iota.org/about-iota/iota-wallet/getting-started)
+
+3. After finding your addresses, go to the [IOTA Wallet Dashboard](https://wallet.iota.org) and check:
+   - **Migration Tab**: For unmigrated Stardust balances  
+     [Migrate Stardust Assets](https://docs.iota.org/about-iota/iota-wallet/migrate-assets)
+   - **Vesting Tab**: For airdropped or locked funds from the October 2024 upgrade  
+     [Manage Vesting and Time-Locked Assets](https://docs.iota.org/about-iota/iota-wallet/vesting)
+
+## 15. I'm Sure My Funds Are in a Specific Address But They’re Not Showing
+
+1. Connect that address to the [Wallet Dashboard](https://wallet.iota.org).
+2. Navigate to the **Migration** tab and copy your legacy address.
+3. Paste it into the [Archived Tangle Explorer](https://explorer.iota.org/mainnet) to check past balances.
```

Comment:
This is not the archive explorer, please use https://archive.explorer.iota.works/

Reply:
lets better use https://archive.explorer.iota.org/, the other is the staging url, thanks